k8s/values.yml

```diff
@@ -10,8 +10,10 @@ service:
 
 ingress:
   enabled: true
-  domain: example.pennlabs.org
+  domain: example.com
 
 secrets:
   enabled: false
-  secretEnv: example-django
+  secretEnv: {{ project_name }}
```

Comment:
Same issue with the docker image. Not sure we want this to always be the same as project name.

Reply:
similar to comment above, don't know if it's a bad thing for simplicity's sake to start a new convention

Reply:
I think this is a good convention to have.

Reply:
Same here.